build-logic/src/main/kotlin/polaris-root.gradle.kts

```diff
@@ -37,7 +37,7 @@ if (!project.extra.has("duplicated-project-sources")) {
   spotless {
     kotlinGradle {
       ktfmt().googleStyle()
-      licenseHeaderFile(rootProject.file("codestyle/copyright-header-java.txt"), "$")
+      // licenseHeaderFile(rootProject.file("codestyle/copyright-header-java.txt"), "$")
```

Comment:
Accidental?

Reply:
JB explained to me that this is to prevent spotlessApply from changing the headers *back* to ASF from dremio. We'll need to figure out how to get the plugin to allow for multiple headers.

Reply:
It's intentional: with that, `spotless:apply` "overwrites" all header with ASF header, which is wrong as we have to keep the original header in case of code copy.

Reply:
@jbonofre mind bringing this back?
There is `targetExcludeIfContentContains/Regex`